esx_service/utils/auth_api.py

```diff
@@ -785,9 +852,8 @@ def check_privilege_parameters(privilege):
 
     return None
 
-
 @only_when_configured()
-def _tenant_access_add(name, datastore, allow_create=None, default_datastore=False,
+def _tenant_access_add(name, datastore, allow_create=None,
```

Comment:
An full access privilege to "default_datastore" will be created during "_tenant_create" automatically.
So, it later user try to add the privilege to "default_datastore", it should return with "PRIVILEGE_ALREADY_EXIST".
User can modify this privilege through "_tenant_access_set".